scripts/zkai_native_attention_mlp_lifting_ablation_gate.py

```
@@ -0,0 +1,750 @@
+from __future__ import annotations
+
+import argparse
+import copy
+import csv
+import hashlib
+import io
+import json
+import os
+import pathlib
+import stat
+from typing import Any, Callable
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)